packages/block-library/src/file/inspector.js

```diff
@@ -41,58 +44,109 @@ export default function FileBlockInspector( {
 		<>
 			<InspectorControls>
 				{ href.endsWith( '.pdf' ) && (
-					<PanelBody title={ __( 'PDF settings' ) }>
-						<ToggleControl
-							__nextHasNoMarginBottom
+					<ToolsPanel
+						label={ __( 'PDF settings' ) }
+						resetAll={ () => {
+							changeDisplayPreview( true );
+							changePreviewHeight( 600 );
+						} }
+						dropdownMenuProps={ dropdownMenuProps }
+					>
+						<ToolsPanelItem
 							label={ __( 'Show inline embed' ) }
-							help={
-								displayPreview
-									? __(
-											"Note: Most phone and tablet browsers won't display embedded PDFs."
-									  )
-									: null
-							}
-							checked={ !! displayPreview }
-							onChange={ changeDisplayPreview }
-						/>
-						{ displayPreview && (
-							<RangeControl
+							isShownByDefault
+							hasValue={ () => !!! displayPreview }
```

Comment:
It is not common practice in this repository to use `!!!`. It maybe be more difficult to read. Is there a specific reason why this was chosen?

Reply:
Or is it a typo?

Reply:
Sorry, I think I made a typo.